### PR TITLE
docs & schema updates

### PR DESCRIPTION
**Pull request by kgvarun**
docs structure & schema yaml updates

### CODE REVIEW

**Review by kgvarun — COMMENTED** [review]
docs + schmea

**Review by kgvarun — COMMENTED** [review]
merging into 3.0.0

File: README.md [hunk 1]
```diff
@@ -1,520 +1,362 @@
...
+##### 0 Pre-requisites
 
-**Note** The following configuration parameters are supported for customisation through Pod Annotations in addition to logSource,
- - oracle.com/oci_la_log_group_id => to use custom logGroupId (oci_la_log_group_id)
- - oracle.com/oci_la_entity_id => to use custom entityId (oci_la_entity_id)
+* Workstation or OCI Cloud Shell with access configured to the target k8s cluster.
+* Helm ([Installation instructions](https://helm.sh/docs/intro/install/)).
 
-#### Through customLogs section
+##### 1 Download helm chart
```
[the remark this answers is not in the document]
Comment (santhoshkvuda) [reply README.md:90]: We can add this only after helm repo url is available from Github pages. The instructions will be updated with a new PR, once V3 lands into main and Github actions tested to create the helm repo automatically.

File: README.md [hunk 2]
```diff
@@ -1,520 +1,362 @@
...
-The following example demonstrates a container customLogs configuration
-```
-      #custom-id1:
-         #path: /var/log/containers/custom*.log
-         #ociLALogSourceName: "Custom1 Logs"
-         #multilineStartRegExp:
-         # Set isContainerLog to false if the log is not a container log (/var/log/containers/*.log). Default value is true.
-         #isContainerLog: true
-```
+* Create override_values.yaml, to override the minimum required variables in values.yaml.
```
[the remark this answers is not in the document]
Comment (santhoshkvuda) [reply README.md:97]: We don't want to suggest Prod, Demo, Test etc., We recommend them to use a new override values yaml for any type of cluster. Name is an example only and don't want to link to an environment at present.

File: README.md [hunk 3]
```diff
@@ -1,520 +1,362 @@
...
+  * Select the region and compartment where you want to deploy the stack.
+  * Follow the on-screen prompts and instructions to create the stack.
+  * After creating the stack, click Terraform Actions, and select Plan.
+  * Wait for the job to be completed, and review the plan.
+  * To make any changes, return to the Stack Details page, click Edit Stack, and make the required changes. Then, run the Plan action again.
+  * If no further changes are necessary, return to the Stack Details page, click Terraform Actions, and select Apply.
+  
+</details>    
+  
+#### Kubectl
```
[the remark this answers is not in the document]
Comment (santhoshkvuda) [reply README.md:179]: Kubectl based installation instructions have reference to helm installation instructions and thus moving this different page means, users need to move around these pages multiple times. Hence, just modified to collapse the instructions.

File: README.md [hunk 4]
```diff
@@ -1,520 +1,362 @@
...
+      
+##### Have customisations to the existing chart or values.yaml
+
+If you have modified values.yaml provided in helm chart directly, we recommend you to identify all the changes and move them to override_values.yaml and follow the instructions provided in install or upgrade sections under [this](#helm). We recommend you to use override_values.yaml for updating values for any variables or to incorporate any customisations on top of existing values.yaml.
+  
+If you are already using a separate values.yaml for your customisations, you still need to compare 2.x vs 3.x variable heirarchy and make the necessary changes accordingly. 
+  
+##### Example 1: Using docker runtime instead of default runtime (cri)
+  
+  **2.x**
```
[the remark this answers is not in the document]
Comment (santhoshkvuda) [reply README.md:287]: Modified to collapse at present.

**Review by palukuri — APPROVED** [review]

**Pull request merged**